cmd.go

```diff
@@ -0,0 +1,14 @@
+package main
+
+import (
+	"github.com/flynn-archive/go-shlex"
+	"github.com/pkg/errors"
+)
+
+// parseCmd is a wrapper function for the underlying shell lexer implementation,
+// it splits a commands string into argv slice based upon shell quoting and escaping
+// rules.
+func parseCmd(cmd string) ([]string, error) {
+	v, err := shlex.Split(cmd)
+	return v, errors.Wrap(err, "shlex: failed to parse command")
```

Comment:
What's the value is parsing the shell command, as opposed to just passing it through?  Are you trying to find more errors, and fail faster?